CMakeLists.txt

```diff
@@ -89,6 +89,14 @@ know it to be required.]=] OFF)
 cmake_dependent_option(OPENVDB_DISABLE_BOOST_IMPLICIT_LINKING
   "Disable the implicit linking of Boost libraries on Windows" ON "WIN32" OFF)
 option(USE_CCACHE "Build using Ccache if found on the path" ON)
+option(USE_STATIC_DEPENDENCIES [=[
+Only search for and use static libraries. If OFF the shared versions of requried libraries are prioritised, falling
+back to static libraries. Forcing individual static dependencies can be enabled by setting XXX_USE_STATIC_LIBS
+to ON, where XXX is the package name. On Windows, this has the important function of changing the default C Runtime
+Library (CRT) linkage to MT from MD for all projects.]=] OFF)
+cmake_dependent_option(USE_WINDOWS_STATIC_RUNTIME
+  "Causes all projects to use the multithread, static version of the C run-time library. Defaults to the value "
+  "of USE_STATIC_DEPENDENCIES." ${USE_STATIC_DEPENDENCIES} "WIN32" OFF)
```

Comment:
Turns out this doesn't work, though if `USE_STATIC_DEPENDENCIES` is ON, `USE_WINDOWS_STATIC_RUNTIME` should really be ON too. I'll update this PR to set `USE_WINDOWS_STATIC_RUNTIME` to ON, with an explicit message